### DISCUSSION TITLE
Error in payment - roboshop infra

### DISCUSSION THREAD

**Discussion by davedevops109** [learndevopsonline/learndevopsonline#1455] [Q&A] [answered]
Hi Raghu,

Furthermore, I got this error in payment component, please advise me

workstation ip : Public ip : 3.238.105.208

![image](https://github.com/learndevopsonline/learndevopsonline/assets/122048247/70ebe246-8440-4b65-b6f1-e39dc0f68470)

**Comment by sreedharm07** [comment, marked as answer]
Hi, @davedevops109 
Error is parameters not found in parameter store.  here i see you are calling  parameters fron payment.service but in your repos i dont find the parameters repo (to create parameters). may be you created parameters from UI.  but the error may be parameters was not created properly or not calling parameters properly. please chek parameters. hoping it will help you
